.github/workflows/ci.yml

```diff
@@ -12,13 +12,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [14.x,16.x,18.x,20.x]
+        node: [14.x, 16.x, 18.x, 20.x]
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node }}
       - run: npm i
-      - run: npm run coverage
+      - run: npm run test
```

Comment:
prettier.

Reply:
Sorry what does that mean? Previously it was running tests with coverage here, now just tests without coverage. Just double checking if that's intended and/or coverage is not needed/used on CI?

Reply:
good point, thx. i proc coverage unconditionally now. will tidy up the scripts